libs/core/src/lib/dialog/dialog.component.ts

```diff
@@ -98,6 +99,14 @@ export class DialogComponent extends DialogBase implements OnInit, OnChanges, Af
         }
     }
 
+    /** @hidden */
+    @ContentChild(DialogTitleDirective)
+    set dialogTitle(title: DialogTitleDirective) {
+        if (title) {
+            this.dialogConfig.ariaLabelledBy = title.titleId;
```

Comment:
wouldn't this overwrite any values that `this.dialogConfig.ariaLabelledBy` previously had? Or are we expecting the user not to use both auto labelling and `ariaLabelledBy` together?